README.md

```diff
@@ -72,7 +72,7 @@ $ rustup override set nightly
  5. Run `cargo new my_bot`, enter the directory and put these lines into your `Cargo.toml`:
 ```toml
 [dependencies]
-teloxide = { version = "0.5", features = ["macros"] }
+teloxide = { version = "0.5", features = ["macros", "auto-send"] }
```

Comment:
```suggestion
teloxide = { version = "0.5.3", features = ["macros"] }
```

Given that we have `auto-send` [specified by default](https://github.com/teloxide/teloxide/blob/8ac3c388d17329de8489f16515e38a94a1a06dfc/Cargo.toml#L28) in our `Cargo.toml`, your Cargo should has just tried to use the wrong version of teloxide. Let me know if v0.5.3 works for you.

Reply:
```
reinaldo@localhost[13:27:05] [~/projetos/reinaldorauch/reminder-bot] [main *]
-> % cargo run
   Compiling reminder-bot v0.1.0 (/home/reinaldo/projetos/reinaldorauch/reminder-bot)
error[E0599]: no method named `auto_send` found for struct `teloxide::Bot` in the current scope
 --> src/main.rs:9:29
  |
9 |   let bot = Bot::from_env().auto_send();
  |                             ^^^^^^^^^ method not found in `teloxide::Bot`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `reminder-bot` due to previous error
reinaldo@localhost[13:27:08] [~/projetos/reinaldorauch/reminder-bot] [main *]
-> % cargo tree | grep teloxide
├── teloxide v0.5.3
│   ├── teloxide-core v0.3.4
│   ├── teloxide-macros v0.4.1 (proc-macro)
```

Cargo.toml:

```
reinaldo@localhost [13:28:49] [~/projetos/reinaldorauch/reminder-bot] [main *]
-> % cat Cargo.toml
[package]
name = "reminder-bot"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
teloxide = { version = "0.5.3", features = ["macros"] }
log = "0.4"
pretty_env_logger = "0.4.0"
tokio = { version =  "1.8", features = ["rt-multi-thread", "macros"] }
```

Reply:
```
reinaldo@TRZPG100596 [13:29:44] [~/projetos/reinaldorauch/reminder-bot] [main *]
-> % rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/reinaldo/.rustup

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu (default)
nightly-x86_64-unknown-linux-gnu

active toolchain
----------------

nightly-x86_64-unknown-linux-gnu (directory override for '/home/reinaldo/projetos/reinaldorauch/reminder-bot')
rustc 1.58.0-nightly (e269e6bf4 2021-10-26)
```

Reply:
I don't think this change (`auto-send` by default) has been released yet.

https://github.com/teloxide/teloxide/blob/178937f9d7d6b71234c35131c47d0bd0b476ad90/Cargo.toml#L28
![image](https://user-images.githubusercontent.com/38225716/142269519-6d3ecf94-7921-45ae-872c-1be52212457e.png)

Reply:
It's strange because @WaffleLapkin's commit to [enable `auto-send`](https://github.com/teloxide/teloxide/commit/56e7b6feb226ca6dc2bb3bf5eb965ae23b8cb60a#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542) is dated Oct. 10, and the v0.5.3 release was published Oct. 25.

@WaffleLapkin any thoughts? Anyway, since this change is still unpublished, I'm going to merge this PR as a temporary workaround.

Reply:
`v0.5.3` release was a backport of a bug fix, see #464.

I think this is a general issue with our current model with `dev` being the default branch - examples are shown by default for the `dev` version which may differ from the latest released one.

Reply:
Okay, I will create an issue about the branch model these weekends. Now I'll just merge this PR as-is.

Reply:
ping @Hirrolot, I think you've forgotten to create an issue